controllers/configurationpolicy_controller.go

```diff
@@ -188,6 +203,16 @@ func (r *ConfigurationPolicyReconciler) PeriodicallyExecConfigPolicies(freq uint
 	}
 }
 
+// handlePolicyWorker is meant to be used as a Go routine that wraps handleObjectTemplates.
+func (r *ConfigurationPolicyReconciler) handlePolicyWorker(
+	policyQueue <-chan *policyv1.ConfigurationPolicy, wg *sync.WaitGroup,
+) {
+	for policy := range policyQueue {
+		r.handleObjectTemplates(*policy)
+		wg.Done()
```

Comment:
This works, but it's still more counting than necessary. I was thinking the WaitGroup would count the workers, not the work items. So the `wg.Done()` would go after the loop here, and the end of the loop occurs when the channel was closed (after all the policies are added to it).